src/List/ListItem.js

```diff
@@ -676,16 +687,16 @@ class ListItem extends Component {
           simpleLabel ? this.createLabelElement(styles, contentChildren, other) :
           disabled ? this.createDisabledElement(styles, contentChildren, other) : (
             <EnhancedButton
-              containerElement="span"
               {...other}
+              containerElement={containerElement}
```

Comment:
I would move `containerElement={containerElement}` before the ` {...other}` just for explicitness. Users should be able to overrides everything.